apps/rehabstod/src/store/slices/settings.slice.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
```

Comment:
The local state has introduced a bug where I can enter values and click "cancel" but the values still show up in the modal if I open it again. It does however not save it as the user's settings. We need to reset the state to the saved values if we close the modal.

Reply:
Good catch! added a reset dispatch, should work as expected now.